common/test/acceptance/pages/lms/discussion.py

```diff
@@ -727,10 +727,11 @@ def get_new_post_preview_value(self, selector=".wmd-preview > *"):
         self.wait_for_element_visibility(selector, "WMD preview pane has contents", timeout=10)
         return self.q(css=".wmd-preview").html[0]
 
-    def get_new_post_preview_text(self):
+    def get_new_post_preview_text(self, selector=".wmd-preview > div"):
```

Comment:
What's the reason for moving this to an argument?

Reply:
First, the 'div' is rendered when we use block mathjax ($$ $$). For inline, the 'p' tag is rendered inside the wmd-preview. By allowing a default value,  we can easily add a test case for inline mathjax, by providing a selector from the test case, if needed.

Reply:
Correct, are you providing this `selector` from any test at the moment?

Reply:
Not at the moment, but I was considering adding a new test case for the Editor, just to confirm flaky behavior is no longer there.

Reply:
Ok.